datasets/species_800/species_800.py

```diff
@@ -0,0 +1,142 @@
+         volume={8},
+         number={6},
+         pages={e65390},
+         year={2013},
+         publisher={Public Library of Science}
+}
+"""
+
+_DESCRIPTION = """\
+We have developed an efficient algorithm and implementation of a dictionary-based approach to named entity recognition,
+which we here use to identifynames of species and other taxa in text. The tool, SPECIES, is more than an order of
+magnitude faster and as accurate as existing tools. The precision and recall was assessed both on an existing gold-standard
+corpus and on a new corpus of 800 abstracts, which were manually annotated after the development of the tool. The corpus
+comprises abstracts from journals selected to represent many taxonomic groups, which gives insights into which types of
+organism names are hard to detect and which are easy. Finally, we have tagged organism names in the entire Medline database
+and developed a web resource, ORGANISMS, that makes the results accessible to the broad community of biologists.
+"""
+
+_HOMEPAGE = "https://species.jensenlab.org/"
+_URL = "https://drive.google.com/u/0/uc?id=1OletxmPYNkz2ltOr9pyT0b0iBtUWxslh&export=download/"
+_BIOBERT_NER_DATASET_DIRECTORY = "s800"
+_TRAINING_FILE = "train.tsv"
+_DEV_FILE = "devel.tsv"
+_TEST_FILE = "test.tsv"
+
+
+class Species800Config(datasets.BuilderConfig):
+    """BuilderConfig for Species800"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for Species800.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(Species800Config, self).__init__(**kwargs)
+
+
+class Species800(datasets.GeneratorBasedBuilder):
+    """Species800 dataset."""
+
+    BUILDER_CONFIGS = [
+        Species800Config(name="species_800", version=datasets.Version("1.0.0"), description="Species800 dataset"),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "tokens": datasets.Sequence(datasets.Value("string")),
+                    "ner_tags": datasets.Sequence(
+                        datasets.features.ClassLabel(
+                            names=[
+                                "O",
+                                "B",
+                                "I",
+                            ]
+                        )
+                    ),
+                }
+            ),
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        urls_to_download = {
+            "biobert_ner_datasets": _URL,
+        }
+        downloaded_files = dl_manager.download_and_extract(urls_to_download)
+        dataset_directory = f"{downloaded_files['biobert_ner_datasets']}/{_BIOBERT_NER_DATASET_DIRECTORY}"
```

Comment:
```suggestion
        dataset_directory = os.path.join(downloaded_files['biobert_ner_datasets'], _BIOBERT_NER_DATASET_DIRECTORY)
```
use `os.path.join` to make it work on windows as well